libmcount/internal.h

```diff
@@ -442,8 +442,8 @@ struct mcount_disasm_info {
 	struct sym		*sym;
 	unsigned long		addr;
 	unsigned char   	insns[64];
-	int			orig_size;
-	int			copy_size;
+	unsigned int		orig_size;
+	unsigned int		copy_size;
```

Comment:
I don't think it's really needed for this change.

Reply:
a commit for this changes will be too small, how do I handle this cases?

Reply:
@namhyung and without this change, compiler will show warning message like below:
/home/m/git/uftrace_hashmap/arch/x86_64/mcount-insn.c:414:23: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
   if (info->orig_size >= instrument_codesize)
                       ^~

Reply:
Why not making instrument_codesize `int` then?